core/src/components/textarea/textarea.tsx

```diff
@@ -278,6 +302,8 @@ export class Textarea implements ComponentInterface {
       this.value = '';
     }
 
+    this.ionInput.emit(null);
```

Comment:
Minor thing, but does `ion-input` emit undefined here? https://github.com/ionic-team/ionic-framework/blob/feature-7.0/core/src/components/input/input.tsx#L446

When I tested in my browser, the `detail` payload showed up as `null`.

Reply:
So in `ion-input`, it can because of this: https://github.com/ionic-team/ionic-framework/blob/feature-7.0/core/src/components/input/input.tsx#L418

I can open up a separate PR to fix that behavior based on our internal discussions of `InputEvent | null` for now.

I will say this pattern doesn't feel intuitive and our thoughts around: 
```
{ value: string, event?: InputEvent | Event }
``` 
appear advantageous before beta.

Reply:
Thanks for the extra info. I don't think this is a blocker right now, but I'll prioritize the research around the `{ value: string, event?: InputEvent | Event }` in the next sprint.